test/provisioner/provision_ibmcloud_initializer.go

```diff
@@ -172,7 +165,7 @@ func initProperties(properties map[string]string) error {
 }
 
 func initVpcV1() error {
-	ibmcloudTrace("initVpcV1()")
+	log.Info("initVpcV1()")
```

Comment:
I think this could have the `Debug` level?

Reply:
Agree